testing/trafficserver/APKBUILD

```diff
@@ -0,0 +1,162 @@
+# Contributor: tcely <trafficserver+aports@tcely.33mail.com>
+# Maintainer: tcely <trafficserver+aports@tcely.33mail.com>
+_upstream="trafficserver"
+pkgname="${_upstream}"
+pkgver="8.0.5"
+pkgrel=0
+pkgdesc="high-performance building block for cloud services"
+url="https://trafficserver.apache.org"
+arch="x86_64"
+license="Apache-2.0"
+options=""
+pkgusers="trafsrvr"
+pkggroups="trafsrvr"
+depends=""
+depends_dev="
+	${pkgname}=${pkgver}-r${pkgrel}
+	${pkgname}-perl=${pkgver}-r${pkgrel}
+	bash
+	curl-dev
+	expat-dev
+	hwloc-dev
+	libexecinfo-dev
+	linux-headers
+	ncurses-dev
+	openssl-dev
+	pcre-dev
+	tcl-dev
+	xz-dev
+	zlib-dev
+	"
+depends_perl="perl"
+depends_static="${pkgname}-dev"
+checkdepends="findutils"
+makedepends="${depends_dev} autoconf automake perl python3"
+install="${pkgname}.pre-install"
+subpackages="
+	${pkgname}-libs-static
+	${pkgname}-libs
+	${pkgname}-dev
+	${pkgname}-doc
+	${pkgname}-perl
```

Comment:
shouldn't it be ::noarch ?

Reply:
It could be, however this includes `/usr/bin/tspush` so I chose not to.

Is there any compelling reason to use `noarch` when there is only one supported architecture anyway?